openslides/motions/models.py

```diff
@@ -409,6 +409,7 @@ def workflow(self):
         """
         Returns the id of the workflow of the motion.
         """
+        # TODO: Rename to workflow_id
```

Comment:
The methods name is "workflow" but it returns the id of a workflow. This is missleading.

This TODO can be fixed by seaching for all places, where workflow is called and change it to workfow_id. Feed free to do it.